packages/document/main-doc/docs/en/guides/basic-features/alias.mdx

```diff
@@ -14,11 +14,11 @@ Modern.js allow aliases in JS and CSS, and the following aliases are built in:
 ```
 
 :::info
-When the optional features is enable, the generator will also add built-in aliases dynamically. For example, when BFF is enabled, the `@api` alias will be added by default.
+When enabling optional features, the `new` command will also dynamically add built-in alias. For example, when enabling BFF, the `@api` alias is added by default.
```

Comment:
the `new` command will also dynamically add built-in alias specific to the features.